ocs_ci/helpers/dr_helpers.py

```diff
@@ -1973,3 +2025,17 @@ def configure_rdr_hub_recovery():
     )
     logger.info("All pre-reqs verified for performing hub recovery")
     return True
+
+
+def uninstall_submariner_on_replacecluster(replacement_cluster):
+    """
+    Uninstall of submariner on replacement cluster
+    """
+    config.switch_acm_ctx()
+
+    # Remove submariner related configs from ACM
+    run_cmd(cmd=f"oc delete SubmarinerConfig submariner " f"-n {replacement_cluster}")
+    run_cmd(
+        cmd=f"oc delete ManagedClusterAddon submariner " f"-n {replacement_cluster}"
```

Comment:
Shouldn't we validate that those resources got deleted successfully?

Reply:
yes, we can have the validation